test/test_ops.py

```diff
@@ -200,7 +200,18 @@ def test_variant_consistency_eager(self, device, dtype, op):
             # Acquires variants to test
             method = op.get_method()
             inplace = op.get_inplace()
-            variants = (v for v in (method, inplace) if v is not None)
+            inplace_ops = [inplace, ]  # list of all inplace ops: inplace variant + alias inplace variants if exist
+            aliases = []
+            for a_op in op.aliases:
+                aliases.append(a_op.op)
+                if a_op.method_variant is not None:
```

Comment:
Because of the filtering that occurs on line 214 I don't think you need to check for None here or on line 209 below. Although that will put None into `inplace_ops` I think that's OK?

Reply:
Yes, you are right about filtering None here and if omitted then `inplace_ops` can be, for example, `[torch_op, None, None]` and yes, it is OK to check `variant` with such list. Just I was thinking that it is not that clean...
I can change as you suggest if you'd like